src/js/modules/accounts/components/SignInModal.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import injectSheet from "react-jss";
+import { Modal } from "react-bootstrap/lib";
+
+import { SignInForm, SignUpForm } from "./forms";
+import { signIn, signUp, closeSignInModal } from "../actions";
+
+const styles = {
+  modalStyle: {
+    width: "50%",
+  },
+};
+
+const SignInModal = ({
+  classes,
+  isSignInModalOpen,
+  closeSignInModal,
+  signIn,
+  signUp,
+  status,
+}) => {
+  if (status.form === "signIn" && status.errors.length === 0) {
```

Comment:
Sign in modal unmounts itself? That's fine...I guess. But you're sure that there won't be an infinite loop where it mounts, then unmounts infinitely?

Reply:
Been tested. Nothing bad os far.